check_stability.py

```diff
@@ -283,6 +286,17 @@ def wptrunner_args(self, root):
             "test_types": ["testharness", "reftest"]
         }
 
+    def extra_setup(self):
+        # https://bugs.chromium.org/p/chromium/issues/detail?id=713947
+        logger.debug("DBUS_SESSION_BUS_ADDRESS %s" % os.environ.get("DBUS_SESSION_BUS_ADDRESS"))
+        if "DBUS_SESSION_BUS_ADDRESS" not in os.environ:
```

Comment:
Hm, I guess I forgot that context. Adding the condition now would make future efforts towards portability easier, but that would be a big job anyway (and one I don't know if we're even considering). I'm fine with leaving it as-is